src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -304,6 +304,39 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
     log should include("Can only bore into modules that are not fully closed")
   }
 
+  it should "not create a new port when source is a port" in {
+    class Baz extends RawModule {
+      val a = IO(Output(Bool()))
+      a := DontCare
+      dontTouch(a)
```

Comment:
Nit: Both the `DontCare` connection and the `dontTouch` are unnecessary given that the test is only looking at CHIRRTL.